<?xml version="1.0"?>
<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/masked_depth/camera_info"/>
    <remap from="image_rect" to="/masked_depth/image"/>
    <remap from="points" to="/obstacle_cloud"/>
  </node>
  <node pkg="three-dimensions_tutorial" type="detection_mask.py" name="detection_mask" output="screen" />
</launch>